thirdparty/SemanticUI-Angular/src/sm-core.js

```diff
@@ -325,7 +325,7 @@
 
 			template: [
 				'<div class="menu">',
-				'  <div class="item" ng-repeat="item in items" data-value="{{ getValue(item) }}" sm-html="label({item:item})"></div>',
+				'  <div class="item" ng-class="{\'disabled\': getDisabled(item)}" ng-repeat="item in items" data-value="{{ getValue(item) }}" sm-html="label({item:item})"></div>',
```

Comment:
Was this auto generated? We won't lose the change if it is updated?

Reply:
Same question for all of the third party stuff?

Reply:
It's not auto generated but we do need to make edits to that pool of third party work. We do the same thing in rainbird-editor for forked 3rd party repos.

Reply:
ok no problem